modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/portlet/action/GetContentDashboardItemInfoMVCResourceCommand.java

```diff
@@ -137,6 +137,10 @@ protected void doServeResource(
 					_toString(contentDashboardItem.getCreateDate())
 				).put(
 					"description", contentDashboardItem.getDescription(locale)
+				).put(
+					"fetchSharingContactsButtonURL",
```

Comment:
this naming is a bit weird to me 🤔 
`fetchManageCollaboratorsButtonURL` could do it better?